### PR TITLE
scan: wire up bridge code tab

### LINKED ISSUE #1302: Wire up Bridge Code Tab on Scan
- Let's comment out the description for now.
- Wire up UI with go & solidity generate function

### PR DESCRIPTION
**Pull request by tansawit**
Fixes: #1302

### CODE REVIEW

File: scan/src/components/oracle-script/OracleScriptBridgeCode.re [hunk 1]
```diff
@@ -141,45 +140,38 @@ module LanguageIcon = {
...
+    let%Opt schema = schemaOpt;
+    let code =
+      switch (language) {
+      | Solidity => Borsh.generateSolidity(schema, "Output") |> Belt_Option.getWithDefault(_, "")
+      | Go => Borsh.generateGo("main", schema, "Output") |> Belt_Option.getWithDefault(_, "")
+      };
+    Some(code);
+  }
+  |> Belt_Option.getWithDefault(_, "");
+};
```
Comment (evilpeach) [inline scan/src/components/oracle-script/OracleScriptBridgeCode.re]: Let's return `codeOpt`, then handle option case on the render.
If code is available, you render it the same way.
If not, you just render the message "Code is not verified."

File: scan/src/components/oracle-script/OracleScriptBridgeCode.re [hunk 2]
```diff
@@ -238,21 +232,22 @@ let make = () => {
...
     <div className=Styles.tableLowerContainer>
       <div className=Styles.vFlex>
         <img src=Images.code className=Styles.codeImage />
-        <Text value="ResultDecoder.sol" size=Text.Lg color=Colors.gray7 />
+        <Text value={getFileNameFromLanguage(~language)} size=Text.Lg color=Colors.gray7 />
       </div>
       <VSpacing size=Spacing.lg />
-      codetest->renderCode
+      {let code = getCodeFromSchema(~schemaOpt, ~language);
+       code->renderCode}
```
Comment (evilpeach) [inline scan/src/components/oracle-script/OracleScriptBridgeCode.re]: Handle `codeOpt` here.

**Review by evilpeach — CHANGES_REQUESTED** [review]

File: scan/src/components/oracle-script/OracleScriptBridgeCode.re [hunk 3]
```diff
@@ -141,45 +140,35 @@ module LanguageIcon = {
...
-      }
-    }|j};
+let getCodeFromSchema = (~schemaOpt, ~language) => {
+  let%Opt schema = schemaOpt;
+  let codeOpt =
+    switch (language) {
+    | Solidity => Borsh.generateSolidity(schema, "Output")
+    | Go => Borsh.generateGo("main", schema, "Output")
+    };
+  codeOpt;
```
Comment (evilpeach) [inline scan/src/components/oracle-script/OracleScriptBridgeCode.re]: Just return it, no need to declare `codeOpt`.

**Review by evilpeach — CHANGES_REQUESTED** [review]

**Review by evilpeach — DISMISSED** [review]
LGTM

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

**Pull request merged**